README.md

```diff
@@ -1,234 +1,184 @@
 
-**Space Travelers' Hub** is a web application for a company that provides commercial and scientific space travel services. This web application allows users to book rockets and join selected space missions.
+I selected the REST Countries API, which provides numeric data about countries around the world. I then built a web application around the API, which has several pages. The home page displays a list of countries that can be filtered by name and the details page.
 
 ## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- JavaScript
+- React
+- Redux
 
 ### Tech Stack <a name="tech-stack"></a>
+- Git and Github
+- VS-code
+- JavaScript
+- HTM
+- CSS
+- React
+- Redux
+- Webpack
 
-<details>
-  <summary>React</summary>
-  <ul>
-    <li><a href="#">React</a></li>
-  </ul>
-</details>
+### Key Features <a name="key-features"></a>
 
-<details>
-  <summary>CSS</summary>
-  <ul>
-    <li><a href="#">CSS</a></li>
-  </ul>
-</details>
+- **Select API**
+- **Build files structure for my React app.** 
+- **Prepare routes and navigation in my app**
+- **display a list of items and filter them**
+- **Create the tests for the application**
+- **Style my components to match the design provided**
+- **Deploy the project and test for final details**
+- **Record a video for my project.**
+- **Create a good README and PR description.**
 
-<details>
-  <summary>Jest testing</summary>
-  <ul>
-    <li><a href="#">Jest testing</a></li>
-  </ul>
-</details>
+
+**React front end framwork**
+- **Git-flow**
+- **Es6 module**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <details>
-  <summary>Redux</summary>
+  <summary>Client</summary>
   <ul>
-    <li><a href="#">Redux</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/React">React</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">JavaScript</a></li>
   </ul>
 </details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-- **React**
-- **API**
-- **Redux**
-- **React-router-dom**
-- **Join or leave missions**
-- **Book rockets**
-- **Profile to see selected missions and rockets booked**
+- [You can check the live demo here](https://countries-cf1d.onrender.com)
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+## 📺 Presentation <a name="presentation"></a>
+- [ScreenRec Presentation](https://www.loom.com/share/7b3d872fbbb8422ab34d7604614f0c61?sid=fc6b615c-2048-4e89-b6b8-2f49f97f2439)
```

Comment:
- [ ] You have done a great job💪 on your video presentation; however🙆‍♀️, you're missing something very important: showing a piece of code👩‍💻 that you're proud of. 🙏Please go ahead and display a piece of code that you're proud of. Remember, in this case, you should show the actual code either from your text editor, GitHub, or any other suitable platform. The key point is to present the actual code, as this will help viewers👨‍👩‍👦‍👦 understand what you have accomplished and showcase your expertise.